### PR TITLE
Chess refactor

### PR DESCRIPTION
**Pull request by wmurphyrd**

### CODE REVIEW

File: src/chess/chess-arbiter.js [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  resetGame() {
+    this.chessEngine = new Chess();
+  },
+
+  handleChatCommand(command) {
+    const id = GameNetwork.getMyId();
+    const profile = window.APP.store.state.profile;
+    switch (command) {
+      case "play":
+        const color = ev.detail[1];
```
Comment (wmurphyrd) [inline src/chess/chess-arbiter.js]: Linter flagged this as `ev is not defined` and sure enough it errors in console when trying this command

File: src/hub.html [hunk 2]
```diff
@@ -1325,6 +1357,16 @@
         </a-entity>
         <a-entity id="scene-preview-node">
         </a-entity>
+        <!-- <a-entity 
+            chess-board="squareSize: 3; hideBoard: false; yCorrections: 0.1 -0.3 -0.6 -0.6 -1.05 -1.3; teleportPlayers: true;"
+            position="-10.3 -2.5 -2"
+            networked="template: #template-waypoint-avatar"
+        ></a-entity> -->
```
Comment (wmurphyrd) [inline src/hub.html]: Do we need to keep this?

**Review by wmurphyrd — COMMENTED** [review]
@quinn-madson this is absolutely beautiful. Love the organization, and the incredibly light touch on upstream hubs files could serve as a model for all hubs extension developers.

I ran auto-lint and pushed it to this branch. I also the the manual linting and proposed once small change in #8 to improve reliability of chess game sync. 

It also looks like the offsets are still set for Alice map - could you change those back to vaporwave until we get the dynamic offsets going?

File: src/chess/chess-arbiter.js (hunk 1 above)
Comment (quinn-madson) [reply src/chess/chess-arbiter.js]: This will be fixed in the chess-refactor-fixes PR awaiting review.

File: src/hub.html (hunk 2 above)
Comment (quinn-madson) [reply src/hub.html]: This will be fixed in the chess-refactor-fixes PR awaiting review.

**Comment by quinn-madson** [comment]
Waiting on chess-refactor-fixes PR to be reviewed and merged into chess-refactor then will merge into vreign.

**Review by quinn-madson — APPROVED** [review]

**Pull request merged**